### DISCUSSION TITLE
Use monospace font for number

### DISCUSSION THREAD

**Discussion by Ensorid** [ppy/osu#31229] [Ideas]
I've noticed that the figures are not all of equal size, as shown in the attached screenshot.
![osu!_Ck5Q62GUT9](https://github.com/user-attachments/assets/978d3337-0336-4f8e-b675-40a16eda1f4e)

**Comment by trustytrojan** [comment]
i think instead of asking for a monospace font for everyone, ask to make it customizable so everyone can have what they want
